src/resources/SP.ts

```diff
@@ -20,6 +20,11 @@ export const BusinessConfigSp = {
       feeCode: null,
       displayName: 'Change Of Directors',
       certifyText: ''
+    },
+    {
+      feeCode: null,
+      displayName: 'Notice of Withdrawal',
+      certifyText: ''
     }
```

Comment:
Do we add it for SP? I added a placeholder here

Reply:
Not needed (nor for GP) because their registrations cannot be FE.

Example: https://dev.create.business.bcregistry.gov.bc.ca/registration-review-confirm?id=TsmwTlWtc9&accountid=2995

Reply:
thank you, removed